rts/motoko-rts/src/constants.rs

```diff
@@ -0,0 +1,11 @@
+use crate::types::{Bytes, Words};
+
+/// Wasm word size. RTS only works correctly on platforms with this word size.
+pub const WORD_SIZE: u32 = 4;
+
+/// Wasm page size (64KiB) in bytes
+pub const WASM_PAGE_SIZE: Bytes<u32> = Bytes(64 * 1024);
+
+/// Wasm heap size (4GiB) in words. Note that `to_bytes` on this value will overflow as 4GiB in
+/// bytes is `u32::MAX + 1`.
```

Comment:
The overflow is a bit of a footgun.  Alternatively we could introduce `MAX_HEAP_SIZE` and make it smaller than `WASM_HEAP_SIZE` by a couple of pages, so that we don't have to worry about accidentally overflowing u32.

Reply:
I'm not sure if we want to reduce heap size to avoid a bug in the RTS. In Rust overflow in arithmetic is checked in debug mode, and we run the tests in both debug mode and release mode, so hopefully any bugs caused by this will be caught before getting merged.